avalon/tools/cbsceneinventory/proxy.py

```diff
@@ -63,6 +63,13 @@ def filterAcceptsRow(self, source_row, source_parent):
         column = self.filterKeyColumn()
         index = model.index(source_row, column, source_parent)
 
+        # Filters below are per version group (top-level entries) so if
```

Comment:
https://github.com/getavalon/core/pull/226/commits/b62bc134aaefba0629c0f71ff912279ba6422a8e tries to reformulate it.